src/plugins/discover/public/context_awareness/profile_providers/logs_data_source_profile/accessors/get_cell_renderers.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { getLogLevelBadgeCell } from '../../../../components/data_types/logs/log_level_badge_cell';
+import type { DataSourceProfileProvider } from '../../../profiles';
+
+export const getCellRenderers: DataSourceProfileProvider['profile']['getCellRenderers'] =
+  (prev) => () => ({
+    ...prev(),
+    'log.level': getLogLevelBadgeCell('log.level'),
+    log_level: getLogLevelBadgeCell('log_level'),
```

Comment:
There is currently `LOG_LEVEL_FIELDS` const for the log level fields in `get_row_indicator_provider.ts`. Can we reuse it here?

Reply:
We might need to also add keyword fields to the list so badges are not missing:

<img width="1092" alt="Screenshot 2024-07-16 at 16 14 41" src="https://github.com/user-attachments/assets/1a4c99ce-7d1c-4974-ad4a-e9644b9e43e3">